ember-cli-build.js

```diff
@@ -17,9 +21,35 @@ module.exports = function() {
   ].map(packageDist);
 
   vendorTrees.push(buildVendorPackage('simple-html-tokenizer'));
-  vendorTrees.push(funnel(path.dirname(require.resolve('handlebars/package')), { 
+  vendorTrees.push(funnel(path.dirname(require.resolve('handlebars/package')), {
     include: ['dist/handlebars.amd.js']
   }));
 
-  return build({ vendorTrees });
-}
\ No newline at end of file
+  let tsAndJs = funnel(__dirname, {
+    include: ['src/**/*.ts', 'src/**/*.js']
+  });
+
+  let templates = funnel(__dirname, {
+    include: ['src/**/*.hbs']
+  });
+
+  let compiledTemplates = new GlimmerTemplatePrecompiler(templates, {
+    rootName: '-application'
```

Comment:
@dgeb This was my interpretation of this thread: https://github.com/glimmerjs/glimmer-application/pull/24#discussion_r107494419

We're hard-coding the main template in `Application` instead of looking it up in the container, so I'm not sure how we would import it into application.ts and also compile it as part of the application-pipeline.